src/autocq.c

```diff
@@ -91,13 +91,17 @@ int auto_cq(void) {
 	    mvprintw(12, 29, "Auto CQ  %-2d ", delayval - 1);
 	    refreshp();
 
-	    usleep(500000); // 500 ms
-	    time_update();
+	    // sleep 10 * 50 = 500ms unless a key is pressed
```

Comment:
Let's handle waiting while sending CQ (line 78) and waiting after CQ uniformly.
I mean

- wait for a total of `message_time` + `delayval`*(1/2 sec)
- in the loop poll for a key each 50 ms
- and call `time_update()` each 500 ms

Reply:
Needs additional logic to handle the countdown display during 'delayval' time.

Anyway it seems sensible to use the same 50ms increment in each case.